t/008_sign_saml.t

```diff
@@ -33,6 +36,34 @@ ok( close XML, "DSA: Signed t/dsa.xml written Sucessfully");
 my $dsaret = $dsasig->verify($dsa_signed_xml);
 ok($dsaret, "XML:Sig DSA: Verifed Successfully");
 
+# SAML metadata
```

Comment:
Can you move this to the bottom of the test.  The SKIP after this added section is related to the section before this.  Otherwise it look finr

Reply:
Sure, done!